### ISSUE TITLE
Improve accessibility of components

### ISSUE THREAD

**Issue by anfeichtinger** [robsontenorio/mary#518]
First of all, thank you to all contributors but especially robsontenorio for creating such an amazing project!

I wanted to ask if there are any plans on improving the components for compliance with the Web Content Accessibility Guidelines (WCAG). In general it would be good to strive for at least WCAG 2 level A or level AA compliance to enable developers to easily create projects for all people. More information on WCAG can be found at [their website](https://www.w3.org/WAI/standards-guidelines/wcag/).

The main improvements I'm looking for here are: 
- keyboard controls for all components where it is possible
- aria labels for screen readers (also reflecting states like if a toggle is enabled)
- using semantic tags in  the correct order

This would be especially helpful for all form elements (e. g. The choices component is not accessible at all).

I should also be able to support this project in implementing parts of it over the summer or discussing possible implementations if things are unclear.

Thanks for your time, have a nice day!

**Comment by robsontenorio** [comment]
Take a look at #461  as a first step. 

I need help from experts.

**Comment by anfeichtinger** [comment]
Okay, I will try to help after my holidays (so in about 2 weeks). I do have some experience in making accessible websites.

**Comment by robsontenorio** [comment]
I really appreciate if you submit one PR per component. Thanks!
